### PR TITLE
msg/async/rdma: Fix compilation warning

### PR DESCRIPTION
**Pull request by Adirl**
Signed-off-by: Sarit Zubakov <saritz@mellanox.com>

### CODE REVIEW

**Comment by Adirl** [comment]
@yuyuyu101 @saritz

File: src/msg/async/rdma/RDMAConnectedSocketImpl.cc [hunk 1]
```diff
@@ -638,10 +638,10 @@ void RDMAConnTCP::cleanup() {
 void RDMAConnectedSocketImpl::notify()
 {
   uint64_t i = 1;
-  int ret;
-
-  ret = write(notify_fd, &i, sizeof(i));
-  assert(ret = sizeof(i));
+  int retVal = write(notify_fd, &i, sizeof(i));
+  if (retVal == -1) {
+     lderr(cct) <<__func__ << " failed to write to fd=" << notify_fd << ", error: " << cpp_strerror(errno) <<dendl;
```
Comment (yuyuyu101) [inline src/msg/async/rdma/RDMAConnectedSocketImpl.cc]: maybe we could assert errno != EAGAIN

**Comment by tchaikov** [comment]
@Adirl  ping?

**Comment by Adirl** [comment]
Sorry for late response,
new patch is attached

**Comment by Adirl** [comment]
anything else needed here?

File: src/msg/async/rdma/RDMAConnectedSocketImpl.cc [hunk 2]
```diff
@@ -657,10 +657,11 @@ void RDMAConnTCP::cleanup() {
 void RDMAConnectedSocketImpl::notify()
 {
   uint64_t i = 1;
-  int ret;
-
-  ret = write(notify_fd, &i, sizeof(i));
-  assert(ret = sizeof(i));
+  int ret = write(notify_fd, &i, sizeof(i));
+  if (ret == -1) {
```
Comment (yuyuyu101) [inline src/msg/async/rdma/RDMAConnectedSocketImpl.cc]: ret = errno to save errno. otherwise lderr may override errno

**Comment by yuyuyu101** [comment]
ping

**Pull request closed without merging**